src/inventory/mod.rs

```diff
@@ -124,6 +127,13 @@ impl Slot {
         self.y = y;
         self.size = size;
     }
+
+    pub fn contains(&self, x: f64, y: f64) -> bool {
```

Comment:
`is_in_bounds`, `is_within` or smth like this are probably better names for this, as `contains` could be interpreted as "contains item" or smth like that if one doesn't look at the types the function takes